crates/binstalk-downloader/Cargo.toml

```diff
@@ -83,14 +109,17 @@ native-tls = [
 ]
 
 # Enable hickory-resolver so that features on it will also be enabled.
-hickory-dns = ["hickory-resolver", "default-net", "ipconfig", "hyper", "once_cell"]
+hickory-dns = ["hickory-resolver", "default-net", "ipconfig", "once_cell"]
 
 # Deprecated alias for hickory-dns, since trust-dns is renamed to hickory-dns
 trust-dns = ["hickory-dns"]
 
+# HTTP3 is temporarily disabled by reqwest.
```

Comment:
is there an issue link to reqwest's http/3 status that we could put here?

Reply:
No, can't find one in reqwest repository.